doc/quickstart/geostyler_quickstart.rst

```diff
@@ -0,0 +1,139 @@
+        .then(style => console.log(style));
+
+
+To connect this style with the UI, we have to store it in a state variable and pass it to
+our ``Style`` component. After defining ``myStyle`` as a state variable, you can use it as follows:
+
+.. code-block:: javascript
+
+    parser.readStyle('<?xml version="1.0" encoding="UTF-8"?><sld:StyledLayerDescriptor xmlns="http://www.opengis.net/sld" xmlns:sld="http://www.opengis.net/sld" xmlns:gml="http://www.opengis.net/gml" xmlns:ogc="http://www.opengis.net/ogc" version="1.0.0"> <sld:NamedLayer> <sld:Name>Default Styler</sld:Name> <sld:UserStyle> <sld:Name>Default Styler</sld:Name> <sld:Title>Gravel_Program_2016</sld:Title> <sld:FeatureTypeStyle> <sld:Name>name</sld:Name> <sld:Rule> <sld:MinScaleDenominator>1.0</sld:MinScaleDenominator> <sld:MaxScaleDenominator>1.0E7</sld:MaxScaleDenominator> <sld:LineSymbolizer> <sld:Stroke> <sld:CssParameter name="stroke">#8000FF</sld:CssParameter> <sld:CssParameter name="stroke-width">3.000</sld:CssParameter> </sld:Stroke> </sld:LineSymbolizer> </sld:Rule> </sld:FeatureTypeStyle> </sld:UserStyle> </sld:NamedLayer> </sld:StyledLayerDescriptor>')
+        .then(style => this.setStyle({myStyle}));
+
+    // ...
+
+    <Style
+        compact={true}
+        style={myStyle}
+    />
+
+The GeoStyler UI should now match the style that was defined as SLD.
+
+To get the edited style as SLD from the ``Style`` component, you can use the ``writeStyle()`` method
+of the geostyler-sld-parser within the ``onStyleChange()`` method of the ``Style`` component.
+
+.. code-block:: javascript
+
+    parser.readStyle('<?xml version="1.0" encoding="UTF-8"?><sld:StyledLayerDescriptor xmlns="http://www.opengis.net/sld" xmlns:sld="http://www.opengis.net/sld" xmlns:gml="http://www.opengis.net/gml" xmlns:ogc="http://www.opengis.net/ogc" version="1.0.0"> <sld:NamedLayer> <sld:Name>Default Styler</sld:Name> <sld:UserStyle> <sld:Name>Default Styler</sld:Name> <sld:Title>Gravel_Program_2016</sld:Title> <sld:FeatureTypeStyle> <sld:Name>name</sld:Name> <sld:Rule> <sld:MinScaleDenominator>1.0</sld:MinScaleDenominator> <sld:MaxScaleDenominator>1.0E7</sld:MaxScaleDenominator> <sld:LineSymbolizer> <sld:Stroke> <sld:CssParameter name="stroke">#8000FF</sld:CssParameter> <sld:CssParameter name="stroke-width">3.000</sld:CssParameter> </sld:Stroke> </sld:LineSymbolizer> </sld:Rule> </sld:FeatureTypeStyle> </sld:UserStyle> </sld:NamedLayer> </sld:StyledLayerDescriptor>')
+        .then(style => this.setStyle({myStyle}));
+
+    // ...
```

Comment:
Sphinx is complaining about code not been js
```
geostyler_quickstart.rst:115: WARNING: Could not lex literal_block as "js". Highlighting skipped.
.
```

So, better remove the `js` on this code-block (or maybe create 2 code blocks with the correct language)

Reply:
I see that its a way too long string, inside the `readStyle` function, and the output becomes one line, because the code block only does highlighting not formatting. My expertise is not javascript, but in C++ we can breakup long strings into different lines. I really hope it can be done on javascript.